html/js/guiutils/Tools.js

```diff
@@ -167,6 +190,19 @@ GTE.UI = (function (parentModule) {
         return this.activePlayer;
     };
 
+    /**
+    * Checks if it is possible to switch to information sets modes. This function is basically
+    * a wrapper that checks that all nodes have a player assigned.
+    * @return {Boolean} True if it is possible to switch to information set mode
+    */
+    Tools.prototype.ableToSwitchToISetMode = function () {
```

Comment:
Would the name "ableToSwitchToISet**Editing**Mode" maybe make it more clear what this fucntion does?